common/hooks/useHotjar.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useContext } from 'react';
+import TogglesContext from '@weco/common/views/components/TogglesContext/TogglesContext';
+
+const useHotjar = () => {
+  const { isHotjarActive } = useContext(TogglesContext);
```

Comment:
Do you think it should be here, or where we instantialise this? I don't really have much of an opinion, just wondering what the thinking was?

I like that it controls itself...

Reply:
Yeah, I wasn't 100% sure. The thinking behind having it as a hook was so that it could be easily instantiated only where it was needed (e.g. just on the search results pages in this instance).

So then I figured that if in the future we wanted to instantiate it in a couple of places, it would be cleaner to be able to wipe it out at the source, rather than in both places it was instantiated.